build/pipelines/templates/codenav-indexer.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  artifactName: 'drop'
+
+jobs:
+- job: CodeNavIndexer
+  displayName: Run Github CodeNav Indexer
+  pool: { vmImage: windows-2019 }

```

Comment:
Does it have to run on Windows? Windows is much slower than Linux at starting up here.

I mean it's fine, but just curious.

Reply:
Unfortunately right now we do have to run on Windows, but we're working on Linux support! We'll send out a mail once we have that ready so you can switch over!